hebiros_advanced_examples/launch/mapping_navigation/omnibase_navigation.launch

```diff
@@ -0,0 +1,46 @@
+<launch>
+  <!-- Arguments -->
+  <arg name="map_file" default="$(find hebiros_advanced_examples)/launch/mapping_navigation/maps/Hebi_map.yaml"/>
```

Comment:
I don't think this is called out anywhere in the documentation, but we should explicitly mention in the README that this is the line we need to change to set the map, and ideally show an example of the call that passes this as an argument as well.